middlewares/verifyToken.js

```diff
@@ -0,0 +1,18 @@
+function verifyToken(req, res, next) {
+  const bearerHeader = req.headers.authorization;
+  if (bearerHeader !== 'undefined') {
+    const token = bearerHeader.split(' ')[1];
```

Comment:
Validate `bearerHeader.split(' ')` before get `[1]`. Can be a bug here if I provide an invalid token

Reply:
If a invalid token gave, how should I handle it ?